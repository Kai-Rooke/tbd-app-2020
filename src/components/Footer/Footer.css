/*Applies styles to both footers*/
.ucpFooter, .MobileFooter {
    border-bottom: 1px solid teal;  
    margin-top: 12%;  
    width: 100%;
    background-image: url('../../images/Footer.png');
    background-position: center;
}

/*Applies styles to desktop footer*/
.ucpFooter{
    height: 150px;
    background-repeat: repeat-x;
}

/*Applies styles to mobile footer*/
.MobileFooter{
    height: 350px;
    background-repeat: no-repeat;
    background-size: cover;
}
