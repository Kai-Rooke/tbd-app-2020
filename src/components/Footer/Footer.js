import React from 'react';
import './Footer.css';
import {isMobile} from 'react-device-detect';


export default class Footers extends React.Component{
    render(){
        const className = isMobile ? "MobileFooter" : "ucpFooter";
        return (
            <footer className={className}/>   
        )
    }
};