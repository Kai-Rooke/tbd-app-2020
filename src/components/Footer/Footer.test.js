import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import Footer from './Footer';

//cleans up 
afterEach(cleanup);

it("Renders Footer", () => {
    const { asFragment } = render(<Footer/>);
    expect(asFragment()).toMatchSnapshot();
});
