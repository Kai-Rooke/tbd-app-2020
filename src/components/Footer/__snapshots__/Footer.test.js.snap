// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Footer 1`] = `
<DocumentFragment>
  <footer
    class="ucpFooter"
  />
</DocumentFragment>
`;
